src/audio-manager.ts

```diff
@@ -0,0 +1,279 @@
+import {_audioContext} from './audio-listener.js';
+
+/**
+ * AudioManager loads and manages audio files from which PlayableNodes are created.
+ *
+ * @example
+ * ```js
+ * async start() {
+ *      this.audio = await audioManager.load('click.wav');
```

Comment:
Use `then` instead of `await`

Reply:
I think that already changed. The changes are on the #23 that still needs to be merged